### ISSUE TITLE
Calling Elixir from .NET?

### ISSUE THREAD

**Issue by tymoor** [svan-jansson/netler#4]
Hi - does this library also allow Elixir to send a message to .NET?

**Comment by svan-jansson** [comment]
No, the current architecture is strictly one-directional: Elixir calls .NET over TCP, and .NET only responds. There's no mechanism for .NET to push messages to an Elixir application unprompted.
